### ISSUE TITLE
Latest version with lot's of deprecations

### ISSUE THREAD

**Issue by christopher5106** [keras-team/keras#12940]
W0611 16:49:22.343930 139742361401152 deprecation_wrapper.py:119] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

W0611 16:49:22.354705 139742361401152 deprecation_wrapper.py:119] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0611 16:49:22.356116 139742361401152 deprecation_wrapper.py:119] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

W0611 16:49:22.377483 139742361401152 deprecation_wrapper.py:119] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

W0611 16:49:22.378576 139742361401152 deprecation_wrapper.py:119] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:133: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

W0611 16:49:22.384115 139742361401152 deprecation.py:506] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
W0611 16:49:23.080871 139742361401152 deprecation_wrapper.py:119] From ./train.py:289: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

Total number of trainable parameters: 1498733
W0611 16:49:23.093107 139742361401152 deprecation_wrapper.py:119] From anaconda3/envs/keras-tf-py36/lib/python3.6/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

**Comment by jvishnuvardhan** [comment]
@christopher5106 As part of performance improvements, removing similar API calls and also to rearrange APIs, it is required to deprecate some of them. But, the log clearly shows how to replace those deprecated APIs with new APIs. 

Please fill the issue template [here](https://github.com/keras-team/keras/issues/new/choose). Could you update them if they are relevant in your case, or leave them as N/A? Along with the template, please provide as many details as possible to find the root cause of the issue.  Thanks!

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by andyDoucette** [comment]
Can this please be opened again?  I'm seeing these warnings with Keras==2.2.4 and tensorflow-gpu==1.14.0.

As far as I understand, there shouldn't be any more information needed.  It's a matter of updating keras to conform to the new tensorflow way of doing things, right?  Most of the warning messages give suggested replacements.

**Comment by christopher5106** [comment]
What I see from these messages, is that Keras 2.2.4 is not compatible with latest versions of Tensorflow.

It has never been very compliant with Theano and CNTK also. To my point of view, it breaks its promise.

**Comment by christopher5106** [comment]
There are two versions of Keras, tf.keras and keras.io

**Comment by RolfChung** [comment]
I imported:

from tensorflow import keras

This gave me:
keras version:  2.2.4-tf

instead of:
import keras
print("keras version: ", keras.__version__)
keras version:  2.2.4

This made the depreciation messages vanish (at least for me).
This maybe coherent with:

Multi-backend Keras and tf.keras:

At this time, we recommend that Keras users who use multi-backend Keras with the TensorFlow backend switch to tf.keras in TensorFlow 2.0. tf.keras is better maintained and has better integration with TensorFlow features (eager execution, distribution support and other).

https://keras.io/

**Comment by caozd999** [comment]
I met with the same issue also when running
```
from keras import models
from keras import layers
```
 I am using python=3.7.2, keras=2.2.4 and tensorflow=1.14.0.
After I modified the code to
```
from tensorflow.keras import models
from tensorflow.keras import layers
```
The warnings disappeared.

**Comment by Hannyul** [comment]
> 
> 
> I met with the same issue also when running
> 
> ```
> from keras import models
> from keras import layers
> ```
> 
> I am using python=3.7.2, keras=2.2.4 and tensorflow=1.14.0.
> After I modified the code to
> 
> ```
> from tensorflow.keras import models
> from tensorflow.keras import layers
> ```
> 
> The warnings disappeared.

Same!! Thank Youuuu
